pages/index.tsx

```diff
@@ -1,9 +1,12 @@
 import type { NextPage } from "next";
-import React, { useState } from "react";
-import Head from "next/head";
+import Company from "./Company-Info-Page";
+
 
 const Home: NextPage = () => {
-  return <></>;
+  return (
+    <>
+      <Company />
```

Comment:
- The application does not start from the `Company` compenent. Delete this line.